js/server/modules/@arangodb/foxx/manifest.js

```diff
@@ -317,9 +323,9 @@ function checkManifest (inputManifest, mount, complainAboutVersionMismatches) {
       console.errorLines(error);
     }
     throw new ArangoError({
-      errorNum: errors.ERROR_INVALID_SERVICE_MANIFEST.code,
+      errorNum: ERROR_INVALID_SERVICE_MANIFEST.code,
```

Comment:
These resulted in the TypeError because "errors" is redefined at the function level.